src/js/techmix_sector.js

```diff
@@ -1,5 +1,4 @@
 import * as d3 from 'd3';
-import { union } from 'd3-array';
```

Comment:
`union` is being used on line 130/129 no?

Reply:
Indeed, however that `union` call is a native method for the Set prototype. 
Check it:

Adding the following console logging to the js there:
``` js
let subgroups = Array.from(subgroups0.union(subgroups1));
console.log(subgroups0.constructor.name);
console.log(Set.prototype.union);
```

and inspecting in the console, I see:
<img width="642" alt="Screenshot 2024-12-02 at 19 19 14" src="https://github.com/user-attachments/assets/e195e26a-2453-4b2e-a5c1-59ec2dbb23f0">

Reply:
And note that if I add the line back:
``` js
import { union } from 'd3-array';
```

and do the same logging, it yields the same source:
<img width="644" alt="Screenshot 2024-12-02 at 19 20 12" src="https://github.com/user-attachments/assets/c3724589-064c-4e39-9baa-5f3a3e2265f2">

We were never calling the `union` method from `d3-array` in the first place

Reply:
Seems like its new: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/union

🤔

Reply:
Might be more backwards compatible to use the `d3` implementation? assuming it does the same thing? 

But I'm not sure how to even force it to actually do/ use that, and feels a bit out of scope for this PR

Reply:
Yeah, it must be a new-ish JS function. D3 used to provide stuff like that before they were available in vanilla JS. I guess luckily the syntax/interface/behavior is sufficiently similar.

Reply:
Well we can monitor it and if things start to seem off, we will know where to look :-)

Reply:
I suspect it is using a vanilla JS Set, and not an old-skool <=D3v4 set/map, but I don't really know the whole evolution of this code anymore, and it's spread out over multiple repos.

https://github.com/RMI-PACTA/pacta-dashboard-svelte/blob/7a24b19a910f9f060adcbe39fc84ae9a510ab991/src/js/techmix_sector.js#L126